app/experimenter/experiments/management/commands/generate_dashboards.py

```diff
@@ -48,14 +44,14 @@ def generate_dashboards(self):
 
                 exp.dashboard_url = dash.public_url
                 exp.save()
-            except RedashClient.RedashClientException as redash_err:
+            except StatisticalDashboard.ExternalAPIError as external_api_err:
```

Comment:
Single letter variables are the tool of the devil, but in Python the ONE place they're generally acceptable is as an exception alias: e

You don't have to, but it's fine if you do.